src/auro-accordion.js

```diff
@@ -106,7 +106,37 @@
     ];
   }
 
+  /**
+   * Internal function that listens for mutation of the content container.
+   * When this occurs, the height of the content is updated to animate correctly.
+   * @private
+   * @returns {void}
+   */
+  handleContentResize() {
```

Comment:
This solution won't work for us at this time because it relies on the event being triggered by the team serving the content. Going with Mutation Observer for now and can revisit if performance is a problem.

Reply:
Would it make sense to add a disconnect as a safeguard / clean up?

```
this.addEventListener('disconnectedCallback', () => {
    mutationObserver.disconnect();
  }, { once: true });
```